src/job.rs

```diff
@@ -50,6 +51,9 @@ pub(crate) struct Job {
     slug: Option<String>,
     wasm: Option<NamedTempFile>,
     toml: Option<NamedTempFile>,
+
+    /// OCI container engine command
+    cmd: OsString,
```

Comment:
Maybe it's an idea to call this `oci_runtime` instead of just `cmd`?

Reply:
Yes, however we want this to be an absolute path for determinism. `oci_runtime` seems to imply "enum" semantics